src/main/java/org/ohdsi/circe/cohortdefinition/builders/MeasurementSqlBuilder.java

```diff
@@ -226,6 +271,18 @@ protected List<String> resolveWhereClauses(T criteria) {
       whereClauses.add(String.format("V.visit_concept_id in (%s)", StringUtils.join(getConceptIdsFromConcepts(criteria.visitType), ",")));
     }
 
+    //Calculation
+    if (criteria.measurementOperand != null) {
+      whereClauses.add(buildNumericRangeClause(
+        MessageFormat.format("(C.value_as_number {0} SC.value_as_number)", criteria.measurementOperand.operator),
+        criteria.measurementOperand.valueAsNumber, ".4f")
```

Comment:
I'm worried that we embed the operator (ie the sql operator) as the string in the payload.

I feel like the operators should be a distinct list of supported operators, and based on the user selection we translate it into the sql operation.